icu4c/source/i18n/unicode/unum.h

```diff
@@ -355,18 +355,15 @@ typedef enum UNumberFormatFields {
      * Approximately sign. In ICU 70, this was categorized under the generic SIGN field.
      * @stable ICU 71
      */
-    UNUM_APPROXIMATELY_SIGN_FIELD,
+    UNUM_APPROXIMATELY_SIGN_FIELD
```

Comment:
Please move the comma back to this line. A trailing enum constant is allowed to have a following comma.